models/DataObject/Concrete.php

```diff
@@ -569,12 +573,12 @@ public function getNextParentForInheritance()
     {
         if ($this->getParent() instanceof AbstractObject) {
             $parent = $this->getParent();
-            while ($parent && $parent->getType() == 'folder') {
+            while ($parent && $parent->getType() === 'folder') {
```

Comment:
Use `Object_Abstract::OBJECT_TYPE_*` constants here and below?

Reply:
Very good idea. Just changed that. Thanks!